README.md

```diff
@@ -164,11 +164,11 @@ PHP Console has multifunctional and smart vars dumper that allows to
 
 ### How to call
 
-**Longest** native debug method call: 
+**Longest** native debug method call:
```

Comment:
What is this change about? Please remove all new line changes. Looks like you're using Windows and pushed CLRF line endings.

Reply:
I am sorry @barbushin, I did not see your question until today. I think my editor removes trailing spaces. It was not my intention to bother you with this.